<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>531</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWidget</string>
  </property>
  <widget class="QPushButton" name="pushButton_close">
   <property name="geometry">
    <rect>
     <x>672</x>
     <y>0</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius:0px;
background-repeat:no-repeat;
border:none;
background-position:center center;
}
QPushButton:hover{
background-color:rgba(230,0,0,5);
}
QPushButton:pressed{
background-color:rgba(95,95,95,100);
border-color:rgba(255,255,255,30);
border-style:insert;
color:rgba(0,0,0,100);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/C:/Users/24332/Desktop/MyMusicPlayer/close.png</normaloff>:/C:/Users/24332/Desktop/MyMusicPlayer/close.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>751</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>741</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_searchname">
       <property name="styleSheet">
        <string notr="true">color:rgb(255,255,255);
background-color:rgba(255,255,50);</string>
       </property>
       <property name="text">
        <string>搜索歌曲名称</string>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_search">
       <property name="styleSheet">
        <string notr="true">color:rgb(255,255,255);
background-color:rgba(255,255,50);</string>
       </property>
       <property name="text">
        <string>搜索</string>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,255,255)
</string>
   </property>
   <property name="text">
    <string>搜索歌曲</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>69</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,255,255)
</string>
   </property>
   <property name="text">
    <string>歌词</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>150</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,255,255)
</string>
   </property>
   <property name="text">
    <string>历史播放</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>170</y>
     <width>751</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>721</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="4,7,4">
     <item>
      <widget class="QListWidget" name="listWidget_searchsongs"/>
     </item>
     <item>
      <widget class="QTextBrowser" name="textBrowser"/>
     </item>
     <item>
      <widget class="QListWidget" name="listWidget_history"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QSlider" name="horizontalSlider_song">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>661</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>450</y>
     <width>61</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,255,255)
</string>
   </property>
   <property name="text">
    <string>00:00</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_voice">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>480</y>
     <width>84</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_voice">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>470</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius:0px;
background-repeat:no-repeat;
border:none;
background-position:center center;
}
QPushButton:hover{
background-color:rgba(230,0,0,5);
}
QPushButton:pressed{
background-color:rgba(95,95,95,100);
border-color:rgba(255,255,255,30);
border-style:insert;
color:rgba(0,0,0,100);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/C:/Users/24332/Desktop/MyMusicPlayer/voice.png</normaloff>:/C:/Users/24332/Desktop/MyMusicPlayer/voice.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>708</x>
     <y>481</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,255,255)
</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>460</y>
     <width>439</width>
     <height>102</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="pushButton_before">
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-radius:0px;
background-repeat:no-repeat;
border:none;
background-position:center center;
}
QPushButton:hover{
background-color:rgba(230,0,0,5);
}
QPushButton:pressed{
background-color:rgba(95,95,95,100);
border-color:rgba(255,255,255,30);
border-style:insert;
color:rgba(0,0,0,100);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/C:/Users/24332/Desktop/MyMusicPlayer/before.png</normaloff>:/C:/Users/24332/Desktop/MyMusicPlayer/before.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_pause">
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-radius:0px;
background-repeat:no-repeat;
border:none;
background-position:center center;
}
QPushButton:hover{
background-color:rgba(230,0,0,5);
}
QPushButton:pressed{
background-color:rgba(95,95,95,100);
border-color:rgba(255,255,255,30);
border-style:insert;
color:rgba(0,0,0,100);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/C:/Users/24332/Desktop/MyMusicPlayer/pause.png</normaloff>:/C:/Users/24332/Desktop/MyMusicPlayer/pause.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_next">
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-radius:0px;
background-repeat:no-repeat;
border:none;
background-position:center center;
}
QPushButton:hover{
background-color:rgba(230,0,0,5);
}
QPushButton:pressed{
background-color:rgba(95,95,95,100);
border-color:rgba(255,255,255,30);
border-style:insert;
color:rgba(0,0,0,100);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/C:/Users/24332/Desktop/MyMusicPlayer/next.png</normaloff>:/C:/Users/24332/Desktop/MyMusicPlayer/next.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_loop">
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-radius:0px;
background-repeat:no-repeat;
border:none;
background-position:center center;
}
QPushButton:hover{
background-color:rgba(230,0,0,5);
}
QPushButton:pressed{
background-color:rgba(95,95,95,100);
border-color:rgba(255,255,255,30);
border-style:insert;
color:rgba(0,0,0,100);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/C:/Users/24332/Desktop/MyMusicPlayer/loop.png</normaloff>:/C:/Users/24332/Desktop/MyMusicPlayer/loop.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
